cmd/agent/common/commands/check.go

```diff
@@ -375,10 +379,9 @@ func Check(loggerName config.LoggerName, confFilePath *string, flagNoColor *bool
 				} else {
 					printMetrics(agg)
 					checkStatus, _ := status.GetCheckStatus(c, s)
-					fmt.Println(string(checkStatus))
-					if saveFlare {
-						writeCheckToFile(c, checkStatus)
-					}
+					statusString := string(checkStatus)
+					fmt.Println(statusString)
+					checkOutput.WriteString(fmt.Sprintln(statusString))
```

Comment:
Wouldn't `checkOutput.WriteString(statusString + "\n")` be simpler/more readable ?

Reply:
Probably! was just trying to be consistent with what gets put to stdout.